docs/secure-connections/sauce-connect/advanced/kgp.md

```diff
@@ -30,19 +30,19 @@ It is developed and maintained by Sauce Labs, and is used to multiplex establish
 
 KGP is preferred over conventional protocols (such as reverse SSH tunnel) for a number of reasons:
 
-    - It’s lightweight
-    - It reconnects when a connection accidentally disconnects
-    - It ensures that all the data is sent and received, even over an unstable or intermittent connection
-    - It provides information about the connection state
+- It’s lightweight
+- It reconnects when a connection accidentally disconnects
+- It ensures that all the data is sent and received, even over an unstable or intermittent connection
+- It provides information about the connection state
 
 ## KGP Packet types
 
 KGP defines the following packet types
 
-    - Data packets carrying HTTP(s) traffic
-    - Control packets, which include:
-    - Connection requests
-    - Keepalive timers
+- Data packets carrying HTTP(s) traffic
+- Control packets, which include:
+- Connection requests
```

Comment:
@waggledans I'd indent these last two bullets since it's like a sub-list

- Data packets carrying HTTP(s) traffic
- Control packets, which include:
  - Connection requests
  - Keepalive timers